lib/src/core/widgets/feactures/cookie_text_field_search.dart

```diff
@@ -47,13 +48,9 @@ class CookieTextFieldSearch extends StatelessWidget {
           borderRadius: BorderRadius.circular(10),
           borderSide: BorderSide.none,
         ),
-        suffixIcon: SvgPicture.asset(
-          'assets/icons/search.svg',
-          fit: BoxFit.scaleDown,
-          colorFilter: ColorFilter.mode(
-            Theme.of(context).colorScheme.onSecondary,
-            BlendMode.srcIn,
-          ),
+        suffixIcon: CookieSvg(
+          path: 'assets/icons/search.svg',
```

Comment:
não seria melhor ja adicionar esses paths num enum pra quando for dar manutenção ficar tudo centralizado nele ?